stories/Autocomplete.stories.js

```diff
@@ -59,18 +59,9 @@ storiesOf('ais-autocomplete', module)
   .add('using vue-autosuggest (multi-index)', () => ({
     template: `
       <div>
-        <ais-autocomplete
-          :indices="[
-            {
-              value: 'airbnb',
-              label: 'ATIS',
-            },
-            {
-              value: 'instantsearch_query_suggestions',
-              label: 'query-suggestions',
-            }
-          ]"
-        >
+        <ais-index index-name="airbnb" />
+        <ais-index index-name="instantsearch_query_suggestions" />
```

Comment:
I don't really understand this. Can you explain what adding these 2 `ais-index` without children does and what it the influence on autocomplete?

Reply:
See "scoped results": algolia/instantsearch.js#3964.